keps/sig-instrumentation/383-new-event-api-ga-graduation/README.md

```diff
@@ -695,6 +695,11 @@ _This section must be completed when targeting beta graduation to a release._
 - Migrate kubelet to use v1beta1 Events - Part 1: [#82677](https://github.com/kubernetes/kubernetes/pull/82677)
 - feat: migrate kube-controller-manager to use v1beta1 events api: [#82834](https://github.com/kubernetes/kubernetes/pull/82834)
 
+New Events API has been graduated to GA in 1.19:
```

Comment:
Can you please remove the 3 points above (migration of leader election, kubelet and kcm)? Those didn't happen so shouldn't be in the history.

Reply:
Done.